### DISCUSSION TITLE
Delete all items from the Backlog

### DISCUSSION THREAD

**Discussion by mawebweb** [Zuehlke/poinz#222] [Ideas]
Hi @xeronimus

We use poinz extensively for our current project. Would be great to have a button to delete all items in the backlog as we always use the same link again for new Sprints.

Best,
Manuel

**Comment by xeronimus** [comment]
Hi Manuel,

thanks for the suggestion :)

Could you specify your idea in more detail:

- is it a "move all stories to the trash"  feature?
- is it a "empty trash" (permanently delete all trashed stories)  feature?
- where could we put that button (in the UI) ?

**Reply by mawebweb** [reply]
Hi Sergio

My original idea was "move all stories to the trash". However, an additional button "empty trash" would also be nice.

UI, what about this?
![Capture](https://user-images.githubusercontent.com/39295312/148171941-e4f0cd12-a734-4b6f-9cb0-a370903a691d.PNG)

**Comment by xeronimus** [comment]
implemented in #223
